docs/extensions/tutorials/_posts/1970-01-05-Installing3rdPartyPackages.md

```diff
@@ -0,0 +1,323 @@
+---
+layout: doc
+permalink: /docs/extensions/tutorials/installing-3rd-party-packages
+title: Installing 3rd Party Packages
+section: Tutorials
+---
+
+# Installing 3rd Party Packages
+<hr />
+
+This tutorial will show you how to install a 3rd party package into an extension and through that into the entire app. For an example we're going to use [react-native-swiper](https://github.com/leecade/react-native-swiper) as a non-native package and [react-native-camera](https://github.com/lwansbrough/react-native-camera) as a native package.
```

Comment:
Before saying what will this tutorial do, I would just repeat that you can do everything with Shoutem, as it's just an architecture on top of React Native, which includes installing 3rd party package as well.